### PR TITLE
Add support for origami values in spring config

### PR DESCRIPTION
**Pull request by terrysahaidak**

### CODE REVIEW

**Comment by jakub-gonet** [comment]
What's the status of this?

**Comment by terrysahaidak** [comment]
Will update it tomorrow, thanks for reminding me.

**Comment by jakub-gonet** [comment]
@terrysahaidak 🏓

File: src/reanimated2/SpringConfig.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  'worklet';
+
+  return {
+    stiffness: stiffnessFromOrigamiValue(tension),
+    damping: dampingFromOrigamiValue(friction),
+  };
+}
+
+export function fromBouncinessAndSpeed(bounciness, speed) {
+  'wroklet';
```
Comment (harterc1) [inline src/reanimated2/SpringConfig.js:23]: typo here

**Review by harterc1 — CHANGES_REQUESTED** [review]

File: src/reanimated2/SpringConfig.js (hunk 1 above)
Comment (terrysahaidak) [reply src/reanimated2/SpringConfig.js:23]: thanks, I'll update it later

**Comment by terrysahaidak** [comment]
working on it right now :)

**Comment by jakub-gonet** [comment]
@terrysahaidak 🏓

**Comment by jakub-gonet** [comment]
@terrysahaidak 🏓 

Do you plan on working on this? Just checking so it doesn't go stale.

**Comment by jakub-gonet** [comment]
@terrysahaidak 🏓

**Comment by mrousavy** [comment]
@jakub-gonet 

image

just so you don't play by yourself

**Pull request closed without merging**